pulp_rpm/app/tasks/publishing.py

```diff
@@ -250,7 +250,7 @@ def get_checksum_type(name, checksum_types):
     return getattr(cr, checksum_type.upper(), cr.SHA256)
 
 
-def publish(repository_version_pk, gpgcheck_options, metadata_signing_service=None,
+def publish(repository_version_pk, gpgcheck_options=None, metadata_signing_service=None,
```

Comment:
:+1:

Reply:
It helps the migration plugin to stay compatible with older versions of pulp_rpm.